src/components/Modals/Send/AddressInput/AddressInput.tsx

```diff
@@ -35,7 +35,13 @@ export const AddressInput = ({ rules }: AddressInputProps) => {
             spellCheck={false}
             autoFocus
             fontSize='sm'
-            onChange={e => onChange(e.target.value.trim())}
+            onChange={e => {
+              if (e.target.value.trim() === '') {
+                setValue(SendFormFields.Address, '')
+              }
+
+              onChange(e.target.value.trim())
+            }}
```

Comment:
While at it, can we also make this whole expression a `useCallback` memoized `handleXYZChange`? see https://github.com/shapeshift/web/pull/2698

Reply:
Not needed anymore there